### PR TITLE
Bug/tcvp 2361

### PR DESCRIPTION
**Pull request by lfdgov**
# Description

This PR includes the following proposed change(s):

- TCVP-2361: NoAppTs not saving -- check other JJ date formats & timezone handling

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts [hunk 1]
```diff
@@ -63,12 +66,13 @@ export class JJDisputeComponent implements OnInit {
...
   ContactType = JJDisputeContactType;
   requireCourtHearingReason: string = "";
+  jjDecisionDateFormattedDate: string;
 
   constructor(
     private formBuilder: FormBuilder,
     private datePipe: DatePipe,
     private authService: AuthService,
-    private jjDisputeService: JJDisputeService,
+    public jjDisputeService: JJDisputeService,
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts:75]:
```suggestion
    private jjDisputeService: JJDisputeService,
```

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.html [hunk 2]
```diff
@@ -194,7 +194,7 @@ <h3>Court Appearance</h3>
           </span>
           <span class="section-grid-cell">
             <p class="section-grid-header">No APP</p>
-            <p class="section-grid-text">{{ courtAppearanceForm?.value.noAppTs | date: "MM/dd/YYYY HH:mm"
+            <p class="section-grid-text">{{ this.noAppTsFormattedDate
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.html:197]:
```suggestion
            <p class="section-grid-text">{{ noAppTsFormattedDate
```

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.html [hunk 3]
```diff
@@ -72,11 +72,11 @@ <h3>Ticket Information / Dispute History</h3>
         </span>
         <span class="section-grid-cell">
           <p class="section-grid-header">Online Submission Date</p>
-          <p class="section-grid-text">{{ lastUpdatedJJDispute.submittedTs | date: "MMM dd, yyyy" }}</p>
+          <p class="section-grid-text">{{ this.submittedDateFormattedDate | date: "MMM dd, yyyy" }}</p>
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.html:75]:
```suggestion
          <p class="section-grid-text">{{ submittedDateFormattedDate | date: "MMM dd, yyyy" }}</p>
```

**Review by Sharkww — APPROVED** [review]

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.html (hunk 3 above)
Comment (lfdgov) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.html:75]: done

**Pull request merged**